pandas/core/frame.py

```diff
@@ -726,13 +726,32 @@ def style(self):
 
     def iteritems(self):
         """
-        Iterator over (column name, Series) pairs.
+        Iterate over DataFrame columns as (column name, Series) pairs.
 
         See also
         --------
         iterrows : Iterate over DataFrame rows as (index, Series) pairs.
         itertuples : Iterate over DataFrame rows as namedtuples of the values.
 
+        Examples
+        --------
+
+        >>> df = pd.DataFrame({'col1': [1, 2], 'col2': [0.1, 0.2]},
+                             index=['a', 'b'])
```

Comment:
Continuation lines need to start with `...` (right below the `>>>`)

Reply:
Also, make sure the `i` in `index` is right below the opening `{` on the line above.